lib/mapUtils.js

```diff
@@ -43,3 +43,156 @@ export async function getProductData(store) {
 export const getMaxWidth = function sync(screenWidth, ebt, seeProducts) {
   return screenWidth - 32 - 12 + (ebt ? -48 : 0) + (seeProducts ? -130 : 0);
 };
+
+export function constructHoursDict(hours) {
+  const daysOfTheWeek = {
```

Comment:
Wondering if we should consolidate some of these things into a file in constants? Lots of repeated code between these different functions.